### DISCUSSION TITLE
New to DSC -- can I use this on prem?

### DISCUSSION THREAD

**Discussion by 100110010111** [PowerShell/DSC#420] [General]
I am brand new to dsc and looking to codify some windows configs. I see dscv3 is actively being worked on where previous version are not. My understanding is that dscv3 is not focused on on-prem infra, is that true or will this work? If not, is dscv1 still worth learning?

**Comment by ThomasNieto** [comment]
DSC in its current form is a stand alone executable allowing it to run anywhere. You can run it on prem or in cloud. One thing to take note DSC v3 isn't GA so it will have code changes and be used for prod scenarios yet.
